CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+
+<br>
+
+## Contributer Getting Started
+
+### SQL Setup
+
+This requires already having a SQL database. If you need to create a SQL database, we recommend one of 2 options:
+- [Local SQL Server running in a Docker container](https://docs.microsoft.com/sql/linux/quickstart-install-connect-docker)
+- [Azure SQL Database](#Create-Azure-SQL-Database).
+
+A primary key must be set in your SQL table before using the bindings. To do this, run the below SQL commands in the SQL query editor.
```

Comment:
Would suggest mentioning that this is a one-time step, they can skip if they already have it done.